schemas/render-submitted-data-document.go

```diff
@@ -0,0 +1,11 @@
+package schemas
+
+//  RenderSubmittedDataDocument struct for render-submitted-data-document topic
```

Comment:
Please use only a single space after the double slash.

Reply:
Remove in [ce005fb](https://github.com/companieshouse/kafka-management-tool/pull/1/commits/ce005fb8926cbc2899d61cf66de233a4e80ce97c)